packages/celotool/src/e2e-tests/governance_tests.ts

```diff
@@ -119,6 +125,25 @@ describe('governance tests', () => {
     return blockNumber % epochSize === 0
   }
 
+  const assertDifferenceCloseTo = (
```

Comment:
Based on the way this is used a signature of `assertAlmostEqual(actual: BigNumber, expected: BigNumber, delta: BigNumber = new BigNumber(10).pow(12).times(5))` would be clearer. The call site might be `assertAlmostEqual(currentBalance.minus(previousBalance), expected)`